AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -91,6 +91,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+mclaren.com##.visible[class*="cookie-alert"][data-component]
```

Comment:
imho too complex
mclaren.com##div[class*="-cookie-alert"] is enough

Reply:
Ok